README.adoc

```diff
@@ -225,18 +229,18 @@ There are several ways to configure an environment variable in a Docker containe
 Create a ConfigMap to configure the app name with the following `kubectl` command.
 [role='command']
 ```
-kubectl create configmap sys-app-name --from-literal name=my-system
+kubectl create configmap sys-app-root --from-literal contextRoot=/dev
 ```
 
-This command deploys a ConfigMap named `sys-app-name` to your cluster. It has a key called `name` with a value of `my-system`. The `--from-literal` flag allows you to specify individual key-value pairs to store in this ConfigMap. Other available options, such as `--from-file` and `--from-env-file`, provide more versatility as to what you want to configure. Details about these options can be found in the https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-configmap-em-[{kube} CLI documentation^].
+This command deploys a ConfigMap named `sys-app-root` to your cluster. It has a key called `contextRoot` with a value of `/dev`. The `--from-literal` flag allows you to specify individual key-value pairs to store in this ConfigMap. Other available options, such as `--from-file` and `--from-env-file`, provide more versatility as to what you want to configure. Details about these options can be found in the https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-configmap-em-[{kube} CLI documentation^].
 
 Create a Secret to configure the credentials that `inventory` will use to authenticate against `system` with the following `kubectl` command.
 [role='command']
 ```
 kubectl create secret generic sys-app-credentials --from-literal username=bob --from-literal password=bobpwd
```

Comment:
The username and password should be different from the original. Otherwise, how do you know the new values were used. I suggest changing to username=alice password=wonderland or something different.

Reply:
After you have the done the changes, you need to update the other places to pass in the right value.